src/index.web.tsx

```diff
@@ -0,0 +1,9 @@
+function ExampleComponent() {
```

Comment:
because of [this babel setting](https://github.com/Expensify/react-native-x-maps/blob/13d2a1614f5c9895d327ffbf1e4c06cf995b6ec0/babel.config.json#L4), `React` is kind of injected during the build.

This is [a feature introduced in React 17](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html). I assumed we're using this pattern in `App` so I configured babel to use this feature.

Anyway, this is pretty common pattern that you don't include `React` in every file unless need to after React 17

Reply:
Oh, that's cool! I don't think we are using that in `App` at all and it always annoyed me 😁

Reply:
for the additional information,[ because of this shared config that we're extending](https://github.com/Expensify/react-native-x-maps/blob/13d2a1614f5c9895d327ffbf1e4c06cf995b6ec0/.eslintrc.js#L9), ESLint doesn't complain missing React in every file (which is correct because we don't need `React`)